omero/sysadmins/unix/server-installation.rst

```diff
@@ -106,15 +106,21 @@ Java SE Runtime Environment (JRE)
 
 If possible, install one of the following packages:
 
-+-----------+---------------------------+
-| System    | Package                   |
-+===========+===========================+
-| Debian    | openjdk-8-jre             |
-+-----------+---------------------------+
-| Homebrew  | N/A (install Oracle Java) |
-+-----------+---------------------------+
-| RedHat    | java-1.8.0-openjdk        |
-+-----------+---------------------------+
+.. list-table::
```

Comment:
These new tables turn weirdly wide. Not a blocker but also maybe not exactly a feature.

Reply:
I was unifying the table since some were wide some not.
I will fix that in a follow up PR
i.e. I will use align left to match the layout in the version requirement page